### DISCUSSION TITLE
First prisma release?

### DISCUSSION THREAD

**Discussion by darkBuddha** [prisma/prisma#7511] [Q&A] [answered]
Hi,
i am wondering when the first version of prisma was released?
Thanks!

**Comment by janpio** [comment, marked as answer]
Which version are you looking for? 
First release ever? First production ready version of Prisma 1? First ever of Prisma 2? First beta of Prisma 2? First GA of Prisma 2?

**Reply by darkBuddha** [reply]
first release ever

**Reply by janpio** [reply]
That would probably be 1.0.2, released to Npm 2018-01-16T17:49:05.
This page lists the first Github releases with descriptions: https://github.com/prisma/prisma1/releases?after=1.2.0, this one being the most interesting I guess: https://github.com/prisma/prisma1/releases/tag/1.0.0

**Reply by darkBuddha** [reply]
Thanks! Looks quite nice, coming from Rails. Giving it a thorough try over the next weeks.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
